README.md

```diff
@@ -8,10 +8,39 @@ This Microservice gets called by the SCA-Wrapper library and has the menu item c
 **Requirement to run the microservice in local**:
 
 1. Clone the repo
-2. Make sure the wrapper data is not running via the service manager. If it is stop it from service manager:
+2. Make sure the wrapper data is not ru>Ô][;p,,]\\,ki\]'ning via the service manager. If it is stop it from service manager:
```

Comment:
Something not right here!

Reply:
I thought I by mistake pressed something in the morning PagerDuty call while drinking glass. but there were no code failure so thought thats not the case. Now I got why :)